src/block/function.ts

```diff
@@ -65,9 +66,13 @@ export default class FunctionBlock extends Block
     public getReturnFromName(name:string):string
     {
         if (/^(is|has)/.test(name)) {
-            return 'boolean';
+            return TypeUtil.getFormattedTypeByName('bool');
         }
 
+        // if (/^get/.test(name)) {
+        //     return 'mixed';
```

Comment:
Can we just take this out of this PR completely, I was debating adding this previously, regardless really it belongs in it's own PR